### ISSUE TITLE
Are the parameters for "300K Random Images " vs. "80m TinyImaget" same.

### ISSUE THREAD

**Issue by JimZAI** [hendrycks/outlier-exposure#22]
Hi, thank you for your insightful work.

I wonder **Are the parameters for "300K Random Images " vs. "80m TinyImaget" same?**
When i reproduce the results, i find that directly replacing "80m TinyImaget" with "300K Random Images " brings significant performance degradation.

Thanks a lot.

**Comment by utkutpcgl** [comment]
Thanks for the contribution.
I am also curious about this.

**Comment by JimZAI** [comment]
Could you please answer our questions. Thanks.

**Comment by andyzoujm** [comment]
Hi, the 300K images were randomly selected after excluding images from the CIFAR, Places, and LSUN classes, as well as images with divisive metadata, from the 80 Million Tiny Images dataset. No further modifications were made. Thank you.

**Comment by JimZAI** [comment]
Thanks for the response.

I would like to ask whether the traning hyperparameters (e.g. learning rate, ...) used for "300K Random Images " and "80m TinyImaget" are the same when performing model training. Because if the hyperparameters are not changed and just "300K Random Images" is used instead of  "80m TinyImaget", the performance of the model degrades very significantly.

**Comment by andyzoujm** [comment]
We didn't tune any hyperparameter on 300K. Can you share some of the results you've obtained?
